app/controllers/users_controller.rb

```diff
@@ -118,7 +118,8 @@ def reservations
   # GET /facilities/:facility_id/users/:user_id/accounts
   def accounts
     # accounts for this facility
-    @account_users = @user.account_users.active
+    facility_accounts = Account.for_facility(current_facility)
+    @account_users = @user.account_users.active.select { |ua| facility_accounts.include?(ua.account) }
```

Comment:
Could this be done as `@user.accounts.for_facility(current_facility)`? It looks like the `User#accounts` relation already has the `active` check on it (well, duplicated), but we'l want to make sure that this don't return deleted accounts.